tests/test_app.py

```diff
@@ -180,6 +233,7 @@ def fake_known_args():
 def test_signal_interception():
     # check that initializing an app calls configure_signal_handlers
     with mock.patch('securedrop_client.app.QApplication'), \
+            mock.patch('securedrop_client.app.socket'), \
```

Comment:
We might as well patch the whole `prevent_second_instance` and not just the socket here. Prevent weird errors during testing.